net/lynx/Makefile

```diff
@@ -0,0 +1,65 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+#
+# Based on https://github.com/kyak/openwrt-packages/tree/master/lynx
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=lynx
+PKG_VERSION:=2.8.9rel.1
+PKG_RELEASE:=1
+
+PKG_LICENSE:=GPL-2.0
+PKG_MAINTAINER:=
+PKG_SOURCE:=$(PKG_NAME)$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=https://invisible-mirror.net/archives/lynx/tarballs/
```

Comment:
There should preferably be at least one more download site in addition to the primary one.
E.g. fossies.org, gentoo or something like those